linera-service/tests/end_to_end_tests.rs

```diff
@@ -2,27 +2,66 @@
 // Copyright (c) Zefchain Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+#[cfg(any(feature = "aws", feature = "rocksdb", feature = "scylladb"))]
 mod common;
 
-use common::INTEGRATION_TEST_GUARD;
-use linera_base::identifiers::ChainId;
-use linera_service::client::{LocalNetwork, Network};
-use std::time::Duration;
+#[cfg(any(feature = "aws", feature = "rocksdb", feature = "scylladb"))]
+use {
+    common::INTEGRATION_TEST_GUARD,
+    linera_base::identifiers::ChainId,
+    linera_service::client::{Database, LocalNetwork, Network},
+    std::time::Duration,
+};
 
+#[cfg(feature = "rocksdb")]
 #[test_log::test(tokio::test)]
-async fn test_end_to_end_reconfiguration_grpc() {
+async fn test_rocks_db_end_to_end_reconfiguration_grpc() {
     let _guard = INTEGRATION_TEST_GUARD.lock().await;
-    test_reconfiguration(Network::Grpc).await;
+    run_reconfiguration(Database::RocksDb, Network::Grpc).await;
 }
 
+#[ignore]
+#[cfg(feature = "aws")]
 #[test_log::test(tokio::test)]
-async fn test_end_to_end_reconfiguration_simple() {
+async fn test_dynamo_db_end_to_end_reconfiguration_grpc() {
     let _guard = INTEGRATION_TEST_GUARD.lock().await;
-    test_reconfiguration(Network::Simple).await;
+    run_reconfiguration(Database::DynamoDb, Network::Grpc).await;
 }
 
-async fn test_reconfiguration(network: Network) {
-    let mut local_net = LocalNetwork::new(network, 4).unwrap();
+#[ignore]
+#[cfg(feature = "scylladb")]
+#[test_log::test(tokio::test)]
+async fn test_scylla_db_end_to_end_reconfiguration_grpc() {
+    let _guard = INTEGRATION_TEST_GUARD.lock().await;
+    run_reconfiguration(Database::ScyllaDb, Network::Grpc).await;
+}
+
+#[cfg(feature = "rocksdb")]
+#[test_log::test(tokio::test)]
+async fn test_rocks_db_end_to_end_reconfiguration_simple() {
+    let _guard = INTEGRATION_TEST_GUARD.lock().await;
+    run_reconfiguration(Database::RocksDb, Network::Simple).await;
+}
+
+#[ignore]
+#[cfg(feature = "aws")]
+#[test_log::test(tokio::test)]
+async fn test_dynamo_db_end_to_end_reconfiguration_simple() {
+    let _guard = INTEGRATION_TEST_GUARD.lock().await;
+    run_reconfiguration(Database::DynamoDb, Network::Simple).await;
+}
+
+#[ignore]
+#[cfg(feature = "scylladb")]
+#[test_log::test(tokio::test)]
+async fn test_scylla_db_end_to_end_reconfiguration_simple() {
+    let _guard = INTEGRATION_TEST_GUARD.lock().await;
+    run_reconfiguration(Database::ScyllaDb, Network::Simple).await;
+}
+
+#[cfg(any(feature = "aws", feature = "rocksdb", feature = "scylladb"))]
+async fn run_reconfiguration(database: Database, network: Network) {
```

Comment:
nit: Please let's be consistent with names: `run_test_db_end_to_end_reconfiguration_simple` (same below)

Reply:
Ok, done.